docs/adr/0009-naming-conventions.md

```diff
@@ -0,0 +1,158 @@
+# Naming conventions
+
+Date: 2020-09-24
+
+## Status
+
+2020-12-02 Proposed
+
+## Component files
+
+### Principles
+
+- Keep it simple, reduce nesting
+- A `src/pages` folder will hold 'root' components and reflect the navigation structure (menu, routes) of the app
+- Components needed only by one 'root' component go into that root component's folder, no extra nesting
+- Components needed by more than one 'root' component go into `src/components`
+- Component folders, files, and auxiliary files, camel cased, starting with upper case
+- Component names and file names will be unique
```

Comment:
I don't agree here: this implies that the small child component names will be needing extra prefixing. It should possible naming files like in the example below: `report/Incident.tsx` and `incidents/incident/Incident.tsx`